xrootd/bind.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package xrootd // import "go-hep.org/x/hep/xrootd"
+
+import (
+	"context"
+
+	"go-hep.org/x/hep/xrootd/encoder"
+	"go-hep.org/x/hep/xrootd/requests/bind"
+)
+
+// Bind a socket to pre-existing session
```

Comment:
The Go convention for documenting code would rather recommend to have something like:

```go
// Bind binds the client's socket to a pre-existing session ID.
func (client *Client) Bind(...) {
```

ie: the name of the thing that we are documenting and then a verb that builds a sentence that reads well.
(I am not a native english speaker either so don't worry: I'll make horrible english sentences as well :P)